pkg/azure/api/api.go

```diff
@@ -217,8 +231,8 @@ func (c *Client) describeVpcs(ctx context.Context) ([]network.VirtualNetwork, er
 	c.limiter.Limit(ctx, "VirtualNetworks.List")
 
 	sinceStart := spanstat.Start()
-	result, err := c.virtualnetworks.ListComplete(ctx, c.resourceGroup)
-	c.metricsAPI.ObserveAPICall("Interfaces.ListAll", deriveStatus(err), sinceStart.Seconds())
+	result, err := c.virtualnetworks.ListAll(ctx)
```

Comment:
Please use [`ListAllComplete`](https://github.com/Azure/azure-sdk-for-go/blob/master/services/network/mgmt/2020-03-01/network/virtualnetworks.go#L572-L586) so it will do pagination automatically.